_includes/sidebar.html

```diff
@@ -12,15 +12,22 @@
     <nav class="sidebar-nav">
       {% for sections in site.contents[page.lang] %}
         {% assign _section = sections[0] | strip %}
+        {% if page.section == _section %}
+          {% assign opened = true %}
+        {% elsif page.section == "home" and forloop.index0 == 1 %}
+          {% assign opened = true %}
```

Comment:
@nscyclone I have tested the pagination thing, it works great. There are two things I would like to comment.
1. It will be better if we can remove the underlining under the arrows. it looks weird.
2. When reading the first lesson, I think it makes more sense to use the word "Home" instead of the Title of the project, but if home hasn't been translated, then use the title of the project (which is Elixir School in English)